backend/app.py

```diff
@@ -1,32 +1,35 @@
-from flask import Flask, request, jsonify, send_from_directory, url_for
+# Good resource to build this out:
+# https://blog.miguelgrinberg.com/post/how-to-create-a-react--flask-project
+
+import util
+import notifications
+from models import Ask, Note, NoteTypes
+from errors import bad_request
+from flask import Flask, request, jsonify, url_for, send_from_directory
+import os
+
+from config import Config
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
-from config import Config
+from database import db
 
 app = Flask(__name__, static_folder='./../frontend/public')
 app.config.from_object(Config)
-db = SQLAlchemy(app)
+#db = SQLAlchemy(app)
```

Comment:
do we need this change with the new database.py file?

Reply:
No, the comment should be removed - just fixed